ukaea/fair-mast
# Add better documentation

To test:
  - Run the notebooks in the docs and make sure they work
  - Run the notebooks through colab
  - Read the notebooks and check they make sense.
  - Code review
